src/main/java/com/whytowait/core/annotations/enums/RequestSource.java

```diff
@@ -0,0 +1,16 @@
+package com.whytowait.core.annotations.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum RequestSource {
```

Comment:
don't provide fieldName here, give the option to put fieldName to check in the annotation itself.

Reply:
What other fields do we need to check, we need only merchantId. i don't think we need user to tell us.

Reply:
The user will not tell us. We will use this in our code; we need to define in which field we expect the value to be checked for resource crud operations.

Reply:
done added a fieldvalue which will tell us where to look for the property.